Simplified/MyBooks/NYPLMyBooksDownloadCenter.m

```diff
@@ -548,16 +548,30 @@ - (void)deleteLocalContentForBookIdentifier:(NSString *const)identifier account:
       if ([book.distributor isEqualToString:OverdriveDistributorKey]) {
         dict = [(NSMutableDictionary *)json mutableCopy];
         dict[@"id"] = book.identifier;
+      } else if ([book.distributor isEqualToString:LCPAudiobooks.distributorKey]) {
+        LCPAudiobooks *lcpAudiobooks = [[LCPAudiobooks alloc] initFor:bookURL];
+        dict = [[lcpAudiobooks contentDictionary] mutableCopy];
+        dict[@"id"] = book.identifier;
       }
       
       [[AudiobookFactory audiobook:dict ?: json] deleteLocalContent];
+      
+      if ([book.distributor isEqualToString:LCPAudiobooks.distributorKey]) {
+        if ([[NSFileManager defaultManager] fileExistsAtPath:bookURL.path]) {
+          NSError *error = nil;
+          [[NSFileManager defaultManager] removeItemAtURL:bookURL error:&error];
+          if (error) {
+            NSString *message = [NSString stringWithFormat:@"%@ %@", book.identifier, book.title];
+            [NYPLErrorLogger logError:error summary:@"Failed to delete audiobook local content" message:message metadata:NULL];
```

Comment:
the message param is going to be deprecated soon. It's preferred to use the `metadata` field instead, because it's more descriptive and it is much easier to inspect on Crashlytics, since you can define your own keys-values.

Can you also change the summary "Failed to delete LCP audiobook local content"?

Reply:
changed and added book.loggableShortString to metadata, I noticed another function logging it

Reply:
oh nice, that's perfect